nano/node/cli.cpp

```diff
@@ -636,7 +636,7 @@ std::error_code nano::handle_node_options (boost::program_options::variables_map
 			nano::daemon_config config{ data_path, network_params };
 			// set the peering port to the default value so that it is printed in the example toml file
 			config.node.peering_port = network_params.network.default_node_port;
-			config.serialize_toml (toml);
+			config.serialize_toml ();
```

Comment:
This is bug. The returned TOML string is discarded., but it should be printed to stdout

Reply:
The point of returning the string was to use it to complete the config of the cli command, with the preamble and the `use_defaults` flag, so that the C++ functionality is maintained.